analysers/formatters/IRVEChecker.py

```diff
@@ -0,0 +1,90 @@
+import re
+
+
+###########################################################################
+
+def remove_trailing_zeros(input_string):
+    """
+    Removes dot and zeros at the end of a floating number typed in string
+    """
+    return str(input_string).replace('.00', '').replace('.0', '')
+
+
+def is_float(str):
+    """
+    Returns True if the input string can be converted to a float, False otherwise.
+    """
+    pattern = r"^-?\d+(\.\d+)?$"
+    return bool(re.match(pattern, str))
+
+
+class IRVE_checker():
```

Comment:
Like other analysers, keep the utilities function into the analyser classes.

Reply:
i moved it in the analyzer file and removed the test file, but i am not sure at all if this is the right way to make it work correctly like other tests